content/09.evolution.md

```diff
@@ -386,7 +386,16 @@ The SARS-CoV-2 pandemic has presented many unprecedented scientific opportunitie
 The rapid identification of the genomic sequence of the virus allowed for early contextualization of SARS-CoV-2 among other known respiratory viruses, and the scientific community has continued to collect, analyze, and disseminate information about the SARS-CoV-2 virus and the associated illness, COVID-19 at previously unimaginable rates [@individual-methods].
 The accessibility of genome sequencing technology has allowed for deep sequencing of the virus to establish a level of viral surveillance that had never before been achieved [@doi:10/d5mg; @doi:10.1038/s41592-022-01444-z; @doi:10.1016/j.lanepe.2021.100210].
 The information obtained from genetic, bioinformatics, and evolutionary analysis has played a significant role in shaping the global pandemic response [@doi:10.1038/s41592-022-01444-z; @doi:10.1038/s41586-021-04188-6; @doi:10/hskp].
-<!-- Wastewater surveillance? Implications for preventing future pandemics? Vaccine response time?-->
+For example, wastewater surveillance has emerged as a potential epidemiological tool to monitor SARS-CoV-2 spread over large regions, complementing clinical surveillance [@doi:10.1016/j.scitotenv.2020.138875; @doi:10.1016/j.coesh.2020.09.006; @doi:10.1016/j.coesh.2020.09.003].
+Humans shed SARS-CoV-2 viral RNA in feces [@doi:10/ggq8zp] that can be detected in wastewater.
+Protocols have been developed to safely and reproducibly isolate and quantify SARS-CoV-2 in samples obtained from wastewater processing plants [@doi:10.1371/journal.pone.0257454; @doi:10.1016/j.coesh.2020.09.003].
+To date, studies show that wastewater surveillance is an effective tool to monitor SARS-CoV-2 spread over large sewersheds [@doi:10.1016/j.scitotenv.2020.138875; @doi:10.1016/j.coesh.2020.09.006; @doi:10.1016/j.coesh.2020.09.003; @doi:10.1039/D1EW00747E]. 
+Indeed, data from a study in New York City indicated that wastewater SARS-CoV-2 detection correlated with clinical detection of infection  [@doi:10.1039/D1EW00747E].
+Similar studies have been conducted in Nevada [@doi:10.1016/j.wroa.2020.100086] and Boston [@doi:10.1128/mSystems.00614-20].
+To date, studies have shown that factors such as temperature, the travel time of wastewater, and diurnal variability may affect detection of SARS-CoV-2 [@doi:10.1016/j.scitotenv.2020.138875; @doi:10.1016/j.wroa.2020.100086].
```

Comment:
```suggestion
To date, studies have shown that factors such as temperature, the travel time of wastewater, and diurnal variability may affect detection of SARS-CoV-2 [@doi:10.1016/j.scitotenv.2020.138875; @doi:10.1016/j.wroa.2020.100086].
Additionaly, wastewater surveillance provides a tool to monitor fluctuations in the viral strains present in a community [@doi:10.1101/2022.02.18.22271188 ; @doi:10.1101/2021.12.21.21268077].
```

Reply:
@RLordan those look good! Maybe something like this?

Reply:
> @RLordan those look good! Maybe something like this?

Yes, superb